src/templates/homepage/HeroSection.jsx

```diff
@@ -24,53 +24,59 @@ const HeroDropdownElem = ({ title }) => (
   </div>
 )
 
-const HeroDropdown = ({ title, options, isActive, toggleDropdown }) => (
-  <div
-    className={`bp-dropdown margin--top--sm ${isActive ? "is-active" : null}`}
-  >
-    <div className="bp-dropdown-trigger">
-      <a
-        className="bp-button bp-dropdown-button hero-dropdown is-centered"
-        role="button"
-        tabIndex="0"
-        aria-haspopup="true"
-        aria-controls="hero-dropdown-menu"
-        onClick={toggleDropdown}
-        onKeyDown={toggleDropdown}
-      >
-        <span>
-          <b>
-            <p>{title}</p>
-          </b>
-        </span>
-        <span className="icon is-small">
-          <i
-            className="sgds-icon sgds-icon-chevron-down is-size-4"
-            aria-hidden="true"
-          />
-        </span>
-      </a>
-    </div>
+const HeroDropdown = ({ title, options }) => {
+  const [dropdownIsActive, setDropdownIsActive] = useState(false)
+  const toggleDropdown = () => setDropdownIsActive((prevState) => !prevState)
+  return (
     <div
-      className="bp-dropdown-menu has-text-left"
-      id="hero-dropdown-menu"
-      role="menu"
+      className={`bp-dropdown margin--top--sm ${
+        dropdownIsActive ? "is-active" : null
+      }`}
     >
-      <div className="bp-dropdown-content is-centered">
-        {options
-          ? options.map((option, index) =>
-              option.title ? (
-                <HeroDropdownElem
-                  key={`dropdown-${index}`}
-                  title={option.title}
-                />
-              ) : null
-            )
-          : null}
+      <div className="bp-dropdown-trigger">
+        <a
+          className="bp-button bp-dropdown-button hero-dropdown is-centered"
+          role="button"
+          tabIndex="0"
+          aria-haspopup="true"
+          aria-controls="hero-dropdown-menu"
+          onClick={toggleDropdown}
+          onKeyDown={toggleDropdown}
+        >
```

Comment:
Think your questions on the template are really valid, but I would suggest not refactoring them in the CMS. 

Currently, the template on the CMS is written these way because this is how it's actually rendered on an Isomer site. By having the CMS template follow the rendered site, we can more closely display a preview of the actual site. 

See below for a side-by-side comparison of the Isomer site html and the CMS templates:

<img width="1551" alt="Screenshot 2022-03-15 at 11 24 24 AM" src="https://user-images.githubusercontent.com/39231249/158299943-d67f33fa-ae91-4590-99ce-16eec7c95f03.png">